### PR TITLE
Added Guren No Yumiya

### LINKED ISSUE #109: Attack On Titan Opening - Guren No Yumiya

### PR DESCRIPTION
**Pull request by DivyeshPuri**
Fixes #109

### CODE REVIEW

File: js/data.json [hunk 1]
```diff
@@ -463,10 +463,16 @@ openings = [
 	},
 		
 	{
-		"link" : "http://67.159.62.2/anime_ost/gintama-op5-single-donten/ettjaclvre/01%20-%20Donten.mp3",
```
Comment (anshumanv) [inline js/data.json]: Why is this line getting removed ?

**Review by anshumanv — COMMENTED** [review]
Check comments

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please put new additions at the end of the array

**Comment by DivyeshPuri** [comment]
@Anshuman-Verma there was a reason i did that.

**Comment by anshumanv** [comment]
What reason ?

**Comment by DivyeshPuri** [comment]
@Anshuman-Verma i think its fine now

**Comment by anshumanv** [comment]
Image ain't good, I'll send link shortly

**Comment by anshumanv** [comment]
Use this https://k43.kn3.net/taringa/D/D/8/D/C/6/t_good/767.jpg

**Review by anshumanv — APPROVED** [review]

**Pull request merged**